jdbc/src/main/resources/interpreter-setting.json

```diff
@@ -115,6 +115,13 @@
         "defaultValue": false,
         "description": "Enable ZeppelinContext variable interpolation into paragraph text",
         "type": "checkbox"
+      },
+      "zeppelin.jdbc.maxConnLifetime": {
+        "envName": null,
+        "propertyName": "zeppelin.jdbc.maxConnLifetime",
+        "defaultValue": -1,
+        "description": "Maximum of connection lifetime in milliseconds. A value of zero or less means the connection has an infinite lifetime.",
```

Comment:
I thought other parameter/config is documented. could you check if you should update the jdbc.md file

Reply:
Updated